examples/1-hello-world/README.md

```diff
@@ -127,7 +127,7 @@ with the following arguments:
   },
   cargo2nix ? builtins.fetchGit {
     url = https://github.com/tenx-tech/cargo2nix;
-    ref = "v0.8.3";
+    ref = "v0.9.0";
```

Comment:
https://github.com/tenx-tech/cargo2nix/blob/master/Cargo.toml#L3

Reminder, need version bump to show up

Reply:
Yep, thanks for the reminder! I decided to preemptively reference this tag in the tutorials in preparation for the v0.9.0 release (which should hopefully be soon, currently blocked on #129 being resolved). The idea is to have the tutorials ready in `master` for the new release before we apply the `v0.9.0` tag and publish it on GitHub.